airbyte-integrations/connectors/destination-redshift/src/main/java/io/airbyte/integrations/destination/redshift/operations/RedshiftS3StagingSqlOperations.java

```diff
@@ -71,14 +71,20 @@ public String getStagingPath(final UUID connectionId,
   }
 
   @Override
-  public void createStageIfNotExists() throws Exception {
+  public String getStageName(String namespace, String streamName) {
+    return "garbage-unused";
```

Comment:
... and I assume we can't just `throw new UnsupportedOperationException` because this method is called unconditionally, but its return value is ignored? >.>

Reply:
Yeah exactly. so have to return some garbage which is unused (so far from tests)